Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.7
+FROM golang:1.8
```

Comment:
@nati , are you ok with removing go1.7 compatibility?

@przemyslaw-dobrowolski-cl , please make sure, we upgrade our build and CI scripts before merging this change. AFAIR, we use go1.7 everywhere.